e2e/utils.go

```
@@ -0,0 +1,157 @@
+	}
+
+	req.Header.Set("x-rh-identity", c.CreateIdentityHeader())
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := c.HTTPClient.Do(req)
+	if err != nil {
+		return nil, nil, fmt.Errorf("request failed: %w", err)
+	}
+
+	responseBody, err := io.ReadAll(resp.Body)
+	resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)